packages/lodestar/src/network/util.ts

```diff
@@ -139,8 +139,10 @@ export async function dialProtocol(
   const timer = setTimeout(() => {
     abortController.abort();
   }, timeout);
+  const signal = abortController.signal;
+  signal?.addEventListener("abort", () => clearTimeout(timer), {once: true});
```

Comment:
I don't think we have to change this function, controller is aborted at timeout so there's no point to do `clearTimeout() ` after a timeout, lets node handle it

Reply:
we should use the abort controller in reqResp, not this local `abortController`

Reply:
> we should use the abort controller in reqResp, not this local abortController
done